GiftedMessenger.js

```diff
@@ -647,6 +669,7 @@ GiftedMessenger.defaultProps = {
 
 GiftedMessenger.propTypes = {
   autoFocus: React.PropTypes.bool,
+  backgroundImage: React.PropTypes.number,
```

Comment:
I thought about that, but I don't think that passing an URL is a good idea since it will be a pretty large background image and the waiting times for it would be very slow considering 3G or 4G connections.
Anyway, that's not up to me, so I will be updating this to also accept URLs :dancer: